src/interface/src/app/services/auth.service.ts

```diff
@@ -104,7 +98,7 @@ export class AuthService {
       );
   }
 
-  validateAccount(token: string) {
+  validateAccount(token: string): Observable<any> {
```

Comment:
can we change this to `Observable<boolean>`? will  make things easier on the long run